packages/components/src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,136 @@
+import React, {
+  ReactElement,
+  ReactNode,
+  createRef,
+  useEffect,
+  useState,
+} from "react";
+import classnames from "classnames";
+import ReactDOM from "react-dom";
+import { AnimatePresence, motion } from "framer-motion";
+import { Text } from "../Text";
+import styles from "./Tooltip.css";
+
+interface TooltipProps {
+  readonly children: ReactElement;
+  /**
+   * Tooltip text
+   */
+  readonly message: string;
+}
+
+export function Tooltip({ message, children }: TooltipProps) {
+  const [direction, setDirection] = useState("above");
+  const [showOnLoad, setShowOnLoad] = useState(false);
```

Comment:
Should we take this out as it's no longer an option?

Reply:
we should. what would i do without you sean

Reply:
Oh, this was just wierdly named state. i need this, but i'll name it properly